core/src/test/java/org/frankframework/http/authentication/OAuthAccessTokenKeycloakTest.java

```diff
@@ -0,0 +1,90 @@
+package org.frankframework.http.authentication;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import com.nimbusds.oauth2.sdk.AccessTokenResponse;
+import com.nimbusds.oauth2.sdk.TokenRequest;
+import com.nimbusds.oauth2.sdk.TokenResponse;
+import com.nimbusds.oauth2.sdk.http.HTTPRequest;
+import com.nimbusds.oauth2.sdk.http.HTTPResponse;
+import com.nimbusds.oauth2.sdk.token.AccessToken;
+
+import dasniko.testcontainers.keycloak.KeycloakContainer;
+import org.apache.http.auth.Credentials;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/**
+ * Tests whether requests for access tokens actually work using a Keycloak test container
+ *
+ * @author erik.van.dongen
+ * @see "https://github.com/dasniko/testcontainers-keycloak"
+ */
+@Testcontainers(disabledWithoutDocker = true)
+@Tag("integration") // Requires Docker; exclude with '-DexcludedGroups=integration'
+public class OAuthAccessTokenKeycloakTest {
+
+	@Container
+	static final KeycloakContainer keycloak = new KeycloakContainer().withRealmImportFile("/Http/Authentication/iaf-test.json");
```

Comment:
Heb je getest dat zonder jouw fixes de test omvalt?

Reply:
Keycloak heeft hier geen last van

Reply:
Ah, dat is dan weer jammer. Maar toch goed om deze test te hebben.

Reply:
Jaa, mooi spul dit